modules/nf-core/centrifuge/build/main.nf

```diff
@@ -0,0 +1,41 @@
+process CENTRIFUGE_BUILD {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::centrifuge=1.0.4"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/centrifuge-core:1.0.4--h5b5514e_0':
+        'quay.io/biocontainers1.0.4--hd03093a_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path conversion_table
+    path taxonomy
```

Comment:
```suggestion
    path taxonomy_tree
```

Looking at the documentation, I am thinking if we should rename taxonomy like that instead